cadc-registry/src/main/java/ca/nrc/cadc/reg/Standards.java

```diff
@@ -162,7 +162,8 @@
     
     public static final URI LOGGING_CONTROL_10 = URI.create("vos://cadc.nrc.ca~vospace/CADC/std/Logging#control-1.0");
 
-    public static final URI PKG_10 = URI.create("vos://cadc.nrc.ca~vospace/CADC/std/Pkg#tar-1.0");
+    // Standard for package requests (ie zip or tar)
+    public static final URI PKG_10 = URI.create("vos://cadc.nrc.ca~vospace/CADC/std/Pkg-1.0");
```

Comment:
@hjeeves Could you please update this to be vault instead of vospace?

Reply:
done